extensions/wikia/MyHome/SpecialWikiActivity.class.php

```diff
@@ -8,151 +8,157 @@ class SpecialWikiActivity extends UnlistedSpecialPage {
-		$wgOut->addExtensionStyle(AssetsManager::getInstance()->getSassCommonURL('extensions/wikia/MyHome/oasis.scss'));
+		$out->addScript( "<script type=\"{$this->wg->JsMimeType}\" src=\"{$this->wg->ExtensionsPath}/wikia/MyHome/WikiActivity.js\"></script>\n" );
+		$out->addExtensionStyle( AssetsManager::getInstance()->getSassCommonURL( 'extensions/wikia/MyHome/oasis.scss' ) );
 
-		wfRunHooks( 'SpecialWikiActivityExecute', array( $wgOut, $wgUser ));
+		wfRunHooks( 'SpecialWikiActivityExecute', [ $out, $user ] );
 
-		$data = $feedProvider->get(50);  // this breaks when set to 60...
+		$data = $feedProvider->get( 50 );  // this breaks when set to 60...
 
 		// FIXME: do it in AchievementsII extension
-		global $wgEnableAchievementsInActivityFeed, $wgEnableAchievementsExt;
-		if((!empty($wgEnableAchievementsInActivityFeed)) && (!empty($wgEnableAchievementsExt))){
-			$wgOut->addExtensionStyle("{$wgExtensionsPath}/wikia/AchievementsII/css/achievements_sidebar.css");
+		if ( ( !empty( $this->wg->EnableAchievementsInActivityFeed ) ) && ( !empty( $this->wg->EnableAchievementsExt ) ) ) {
+			$out->addExtensionStyle( "{$this->wg->ExtensionsPath}/wikia/AchievementsII/css/achievements_sidebar.css" );
 		}
 
 		// use message from MyHome as special page title
-		$wgOut->setPageTitle(wfMsg('oasis-activity-header'));
+		$out->setPageTitle( $this->msg( 'oasis-activity-header' ) );
 
-		$template = new EasyTemplate(dirname(__FILE__).'/templates');
-		$template->set('data', $data['results']);
+		$template = new EasyTemplate( __DIR__ . '/templates' );
+		$template->set( 'data', $data['results'] );
 
-		$showMore = isset($data['query-continue']);
-		if ($showMore) {
-			$template->set('query_continue', $data['query-continue']);
+		$showMore = isset( $data['query-continue'] );
+		if ( $showMore ) {
+			$template->set( 'query_continue', $data['query-continue'] );
 		}
-		if (empty($data['results'])) {
-			$template->set('emptyMessage', wfMsgExt("myhome-activity-feed-empty", array( 'parse' )));
+		if ( empty( $data['results'] ) ) {
+			$template->set( 'emptyMessage', $this->msg( 'myhome-activity-feed-empty' )->parse() );
 		}
 
-		$template->set_vars(array(
+		$template->set_vars( [
 			'showMore' => $showMore,
 			'type' => $this->feedSelected,
-			'wgBlankImgUrl' => $wgBlankImgUrl,
-		));
+			'wgBlankImgUrl' => $this->wg->BlankImgUrl,
+		] );
 
-		$wgOut->addHTML($template->render('activityfeed.oasis'));
+		$out->addHTML( $template->render( 'activityfeed.oasis' ) );
 
 		// page header: replace subtitle with navigation
 		global $wgHooks;
-		$wgHooks['PageHeaderIndexAfterExecute'][] = array($this, 'addNavigation');
+		$wgHooks['PageHeaderIndexAfterExecute'][] = [ $this, 'addNavigation' ];
 
-		if ($wgUser->isAnon()) {
-			$this->getOutput()->setSquidMaxage( 3600 ); // 1 hour
-			$this->getOutput()->tagWithSurrogateKeys(
+		if ( $user->isAnon() ) {
+			$out->setSquidMaxage( 3600 ); // 1 hour
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)